posthog/migrations/0787_migrate_playlist_types.py

```diff
@@ -0,0 +1,39 @@
+from django.db import migrations
+
+
+def migrate_playlist_types(apps, schema_editor):
+    SessionRecordingPlaylist = apps.get_model("posthog", "SessionRecordingPlaylist")
+    SessionRecordingPlaylistItem = apps.get_model("posthog", "SessionRecordingPlaylistItem")
+
+    # 1. Get IDs of playlists that have playlist_items (should be COLLECTION)
+    playlist_ids_with_items = set(
+        SessionRecordingPlaylistItem.objects.filter(playlist__type__isnull=True, playlist__deleted=False)
+        .values_list("playlist_id", flat=True)
+        .distinct()
+    )
+
+    # 2. Update playlists with items to COLLECTION type using chunking
+    playlists_with_items = SessionRecordingPlaylist.objects.filter(
+        id__in=playlist_ids_with_items, type__isnull=True, deleted=False
+    )
+
+    for playlist in playlists_with_items.iterator(chunk_size=100):
+        playlist.type = "collection"
+        playlist.save(update_fields=["type"])
+
+    # 3. Update remaining playlists with null type to FILTERS type using chunking
+    remaining_playlists = SessionRecordingPlaylist.objects.filter(type__isnull=True, deleted=False)
+
+    for playlist in remaining_playlists.iterator(chunk_size=100):
+        playlist.type = "filters"
+        playlist.save(update_fields=["type"])
```

Comment:
since we don't need the `save` signals we can avoid issuing 100 update statements here with something like

```
for playlist in remaining_playlists:
    playlist.type = "filters"

SessionRecordingPlaylist.objects.bulk_update(remaining_playlists, ["type"])
```

that way it's a single query to get the chunk of 100, and then a single sql update to store them